src/test/java/com/ss/ita/rozetka/test/RecentlyViewedProductsTest.java

```diff
@@ -115,14 +112,79 @@ public void verifyProductAddedOnlyOnce() {
                 .openProductTypePage(KITCHEN_APPLIANCES_CATEGORY)
                 .openProductPage(1)
                 .getTitle();
-        var recentlyViewedProductTitle = header
+        var recentlyViewedProductTitles = header
                 .openHomePage()
-                .getRecentlyViewedProductTitle();
-        assertThat(recentlyViewedProductTitle)
+                .getRecentlyViewedProductTitles();
+        assertThat(recentlyViewedProductTitles)
                 .as("Recently viewed product list should contain one product")
                 .hasSize(1);
-        assertThat(recentlyViewedProductTitle)
+        assertThat(recentlyViewedProductTitles)
                 .as("Product in recently viewed product list should be the same as last opened product")
                 .contains(secondOpenedProductTitle);
     }
+
+    @Test
+    @Description(value = "Verifies that opening already opened product moves it to the first position in recently viewed products")
+    @TmsLink(value = "LVTAQC630-60")
+    public void verifyProductMovedToFirstPosition() {
+        var homePage = new HomePage().open();
+
+        var productPage = homePage
+                .openProductCategoryPage(HOUSEHOLD_APPLIANCES)
+                .openProductTypePage(KITCHEN_APPLIANCES_CATEGORY);
+        var isProductTypePageOpened = productPage.isOpened();
+        assertThat(getCurrentUrl())
+                .as("Kitchen appliances category page should be opened")
+                .isEqualTo("https://bt.rozetka.com.ua/tehnika-dlya-kuhni/c435974/");
```

Comment:
this verification could potentially make test unstable if Rozetka's default language is not UA